queries/orderbook/batch_rewards.sql

```diff
@@ -81,17 +81,18 @@ WITH observed_settlements AS (SELECT
                               FROM participation_data
                               GROUP BY participant),
      primary_rewards as (SELECT rpt.solver,
-                                SUM(capped_reward)  as total_reward_eth,
-                                SUM(execution_cost) as total_exececution_cost_eth
+                                SUM(capped_payment) as payment_wei,
+                                SUM(execution_cost) as exececution_cost_wei
```

Comment:
Is the typo here deliberate?

Reply:
Of course not. Just a regular old unintentional typo.